src/components/installation/InstallationLogModal.tsx

```diff
@@ -0,0 +1,137 @@
+import { SSEContext } from "@/context/sse-context";
```

Comment:
Would be nice if we could use an existing component, i.e.: 
https://github.com/raspiblitz/raspiblitz-web/blob/chore/deps-250412/src/layouts/ModalDialog.tsx

And maybe extend/adjust it if needed